README.md

```diff
@@ -104,12 +110,15 @@ This repository comes with out-of-the-box integrations with Google Cloud Logging
 and Managed Prometheus for monitoring. To see your Ray cluster logs:
 
 1. Open Cloud Console and open Logging
-2. Use the following query parameters:
+2. Note the job ID returned by the `ray job submit` API.
```

Comment:
We might need to explain how to do `ray job submit` in this setup. The previous steps tell users to launch workloads using notebooks which does not provide a job id.